connectors/byoei.py

```diff
@@ -175,10 +177,13 @@ def __init__(
         queue,
         index,
         existing_ids,
+        filtering=None,
```

Comment:
Why allow for None here?

Reply:
Good point mixed up best practices for mutable data types and default keyword args. Will change it.